topo_processor/util/checksum.py

```diff
@@ -8,9 +8,10 @@
 CHUNK_SIZE = 1024 * 1024  # 1MB
 
 
-def multihash_as_hex(path: str) -> Union[str, Any]:
+def multihash_as_hex(path: str) -> str:
     file_hash = hashlib.sha256()
     with get_fs(path).open(path, "rb") as file:
         while chunk := file.read(CHUNK_SIZE):
             file_hash.update(chunk)
-    return multihash.to_hex_string(multihash.encode(file_hash.digest(), "sha2-256"))
+    result: str = multihash.to_hex_string(multihash.encode(file_hash.digest(), "sha2-256"))
```

Comment:
`Mypy` was complaining because `to_hex_string` method was not typed as returning a `str`, even if it does return the result of a function typed as returning a `str` (`to_hex_string` returns `hexlify(multihash).decode()` and `decode()` is typed as a `str` return value annotation).
By specified the type for the return value (because we know that `to_hex_string()` always returns a `str`, we force mypy to consider `result` as a `str`.